ee/ucp/admin/install/install-on-azure.md

```diff
@@ -206,13 +200,59 @@ for each virtual machine in the virtual machine scale set.
 
 ## Install UCP 
 
-Use the following command to install UCP on the manager node.
-The `--pod-cidr` option maps to the IP address range that you configured for
-the subnets in the previous sections, and the `--host-address` maps to the
-IP address of the master node.
+### Adjusting the IP Count Value
 
-> Note: The `pod-cidr` range must be within an Azure subnet attached to the
-> host.
+If you have manually attached additional IP addresses to the Virtual Machines (via an ARM Template, Azure CLI or Azure Portal) or you want to reduce the number of IP Addresses automatically provisioned by UCP from the default of 128 addresses, then you should alter the IP Count Value in the UCP Configuration file before installation. If you are happy with 128 addresses per Virtual Machine, proceed to [Installing UCP](#installing-ucp).
+
+If you have manually attached additional IP addresses then your UCP configuration file would be: 
+
+```
+$ cat example-config-1
+[cluster_config]
+  azure_ip_count = "0"
+```
+
+If you want to reduce the IP Addresses dynamically allocated from 128 to a
+custom value (see [Considerations for IPAM
+Configuration](#considerations-for-ipam-configuration) to calculate an
+appropriate value) then your UCP configuration file would be: 
+
+```
+$ cat example-config-2
+[cluster_config]
+  azure_ip_count = "20" # Note this value may be different for your environment
+```
+
+To pre-load this configuration file prior to a UCP installation, connect to the Virtual Machine where you would you like to install UCP and initiate a Swarm. 
+
+```
+$ docker swarm init
+```
+
+Then preload the UCP configuration file as a Swarm Config.
```

Comment:
The section felt choppy to me. We show a bunch of config examples but should take a moment to explain the context of what a config object does. If we need to create one, let's go ahead and instruct them to do a `touch azure-config.toml` or similar and be specific on if they need to take an entire existing config and edit it, or if it's sufficient to just use a 2-liner such as the above's:
```
[cluster_config]
  azure_ip_count = "20"
```

Reply:
@stevenfollis Sorry to be pain but do you mind taking a second look? I've expanded this section out a bit :)